src/components/stakeholder/member-table.tsx

```diff
@@ -203,26 +245,39 @@ export const columns: ColumnDef<TypeGetMembers[0]>[] = [
           <DropdownMenuContent align="end">
             <DropdownMenuLabel>Actions</DropdownMenuLabel>
 
-            <MemberModal
-              title="Update stakeholder"
-              subtitle="Update stakeholder's account information."
-              member={{
-                name: member.user?.name ?? "",
-                email: member.user?.email ?? "",
-                title: member.title ?? "",
-                access: member.access ?? "stakesholder",
-              }}
-            >
-              <span className="relative flex cursor-default select-none items-center rounded-sm px-2 py-1.5 text-sm outline-none transition-colors focus:bg-accent focus:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50">
-                Update
-              </span>
-            </MemberModal>
-
-            <DropdownMenuItem>Reinvite</DropdownMenuItem>
+            {isAdmin && status === "accepted" && (
+              <MemberModal
+                isEditMode
+                membershipId={member.id}
+                title="Update stakeholder"
+                subtitle="Update stakeholder's account information."
+                member={{
+                  name: member.user?.name ?? "",
+                  email: email ?? "",
+                  title: member.title ?? "",
```

Comment:
@dahal should we allow the admin to update a stake holders email ? since this is gonna effect the session of stakeholder because JWT cannot be revoked or updated.

Reply:
@G3root Thats a good point, lets just make the email field disabled for now. We will introduce work email and login email down the line. Login email is useful when an employee is of-boarded but still wants to keep track of their shares/equities.